### PR TITLE
Deepks

### PR DESCRIPTION
**Pull request by wenfei-li**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ project(ABACUS
     LANGUAGES CXX
 )
 
-option(ENABLE_DEEPKS "Enable DeePKS functionality" OFF)
+option(ENABLE_DEEPKS "Enable DeePKS functionality" ON)
```
Comment (caic99) [inline CMakeLists.txt]: Please do not changing the default behavior of abacus.

File: source/module_base/complexmatrix.h [hunk 2]
```diff
@@ -57,6 +57,9 @@ class ComplexMatrix
 	void set_as_identity_matrix(void);
 
 	std::ostream & print( std::ostream & os, const double threshold_abs=0.0, const double threshold_imag=0.0 ) const;		// Peize Lin add 2021.09.08
+
+	ModuleBase::matrix dble(void);
```
Comment (caic99) [inline source/module_base/complexmatrix.h]: What does `dble` mean? No documentation for these functions.

File: source/src_lcao/LCAO_descriptor_old.cpp [hunk 3]
```diff
@@ -2,18 +2,6 @@
 #ifdef __DEEPKS
```
Comment (caic99) [inline source/src_lcao/LCAO_descriptor_old.cpp:2]: If no longer used, why should we keep this file in repo?

File: CMakeLists.txt (hunk 1 above)
Comment (wenfei-li) [reply CMakeLists.txt]: sure

File: source/src_lcao/LCAO_descriptor_old.cpp (hunk 3 above)
Comment (wenfei-li) [reply source/src_lcao/LCAO_descriptor_old.cpp:2]: will clean up in the end; currently keeping this for checking purpose

File: source/module_base/complexmatrix.h (hunk 2 above)
Comment (wenfei-li) [reply source/module_base/complexmatrix.h]: i'm actually not using it anymore, will remove it

File: source/src_lcao/ELEC_cbands_k.cpp [hunk 4]
```diff
@@ -5,6 +5,7 @@
 #include "../src_pw/symmetry_rho.h"
 #include "LCAO_evolve.h"
 #include "dftu.h"
+#include "LCAO_descriptor.h"
```
Comment (dyzheng) [inline source/src_lcao/ELEC_cbands_k.cpp:9]: please add ifdef __DEEPKS for that

File: tests/deepks/Autotest.sh [hunk 5]
```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 # ABACUS executable path
-abacus=/home/fortneu49/abacus-develop/bin/ABACUS.mpi
+abacus=/home/wenfei/abacus-develop/build/abacus
```
Comment (dyzheng) [inline tests/deepks/Autotest.sh]: better turn this default to abacus=abacus

Comment (wenfei-li) [reply tests/deepks/Autotest.sh]: sure

File: source/src_lcao/ELEC_cbands_k.cpp (hunk 4 above)
Comment (wenfei-li) [reply source/src_lcao/ELEC_cbands_k.cpp:9]: ok

**Review by dyzheng — APPROVED** [review]

**Pull request merged**